client/src/components/ModalMotion/ModalMotion.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Typography from 'material-ui/Typography';
+import CSSTransition from "react-transition-group/CSSTransition";
+import './ModalMotion.css';
+import Modal from 'material-ui/Modal/Modal';
+
+const animationTiming = {
+  enter: 400,
+  exit: 1000
+};
+
+/**
+ * Modal with transitions
+ */
+const ModalMotion = props => {
```

Comment:
I don't know the material-ui library, not even what `CSSTransition` component does, but I would call this something like `ModalTransition`, `TransitionedModal`. Sorry for the incomplete review, I'm just watching it over. Hope to have better reviews soon ;)

Reply:
I choice this name because https://github.com/chenglou/react-motion 
------------------------------
U feel free to change it and create PR